files/usr/share/defaults/jeos-firstboot.conf

```diff
@@ -0,0 +1,18 @@
+# Example configuration file for jeos-firstboot
+
+# Valid system locale that will be used in JeOS. If is not set, a
+# dialog box will ask for the computer locale.
+# JEOS_LOCALE='en_US'
+
+# Keyboard layout used after the boot. If is not set, a dialog box
+# will ask for the keyboard layout.
+# JEOS_KEYTABLE='en'
+
+# Local timezone of the node. If is not set, a dialog box will ask for
+# the local timezone.
+# JEOS_TIMEZONE='UTC'
+
+# If is set to a different value than '0', the dialog box for setting
```

Comment:
`"0"` will also skip setting the password.

```suggestion
# If is set to a nonempty value, the dialog box for setting
```

Reply:
True. Done